features/attach_validtoken.feature

```diff
@@ -5,7 +5,7 @@ Feature: Command behaviour when attaching a machine to an Ubuntu Advantage
     @series.trusty
     Scenario: Attach command in a trusty lxd container
        Given a `trusty` lxd container with ubuntu-advantage-tools installed
-        When I attach contract_token with sudo
+        When I attach `contract_token` with sudo
```

Comment:
was this some magic to not run tests on services that didn't have a contract service behind it?

Reply:
Basically behave works on structured language processing in order to properly match a python function to call in features/steps/steps.py. I added the backticks ` bookended around the contract_token string to allow behave to easily see the beginning and end of the structured string format:
    "I attach `blah1` blah blah2" 

This allows behave to match any string of this format to the following structured function in features/steps.steps.py:

@when("I attach `{token_type}` {user_spec}")                                    
when_i_attach_staging_token(context, token_type, user_spec):        

behave splits up my formatted string and calls:
when_i_attach_staging_token(token_type="blah1", user_spec="blah blah2")

If we didn't have backticks in that structured string
@when("I attach {token_type} {user_spec}")

and my test defined
When I attach 1 2 3
behave wouldn't know to pass
token_type="1",user_spec="2 3"  OR  token_type="1 2", user_spec="3"